### ISSUE TITLE
a11y e2e: Edit or Add a device

### ISSUE THREAD

**Issue by johanna-skylight** [CDCgov/prime-simplereport#5053]
### Background
The current Edit or Add device form under the admin role has below a11y violations:

1. 'heading-order'
2. 'label-title-only'
3. 'label'
4. 'page-has-heading-one'

### Action requested
Fix a11y violations listed above and remove the a11y exclusions in the e2e spec #09 (multiplex flows)

### Acceptance Criteria
Verify that the edit or add device form should pass the axe scan successfully and no a11y violations are raised.
Verify that spec 09 that tests the multiplex flows passes all a11y checks without the need of exclusions.

### Additional context
Screenshot of the form
![Screenshot 2023-01-19 at 9.25.53 AM.png](https://images.zenhubusercontent.com/626181d9445ca03988cf710d/9cd52de9-e83c-4adc-a55f-73576afaa893)
